core/badge-urls/make-badge-url.js

```diff
@@ -71,9 +71,43 @@ function staticBadgeUrl({
   return `${baseUrl}/badge/${path}.${format}${suffix}`
 }
 
+function dynamicBadgeUrl({
+  baseUrl,
+  datatype,
+  label,
+  dataUrl,
+  query,
+  prefix,
+  suffix,
+  color,
+  style,
+  format = 'svg',
```

Comment:
I noticed the previous implementation allowed `queryParams` to be passed in as an object along with `rest`, so were those unused/no longer necessary? (I see `style` can now be passed in directly and is then added to the queryParams object)

Reply:
Yea, that's right. We didn't seem to be using that anywhere so I thought it was better to make a more direct API that calls out out the params we are using.